assets/queries/common/passwords_and_secrets_in_infrastructure_code/query.rego

```diff
@@ -0,0 +1,290 @@
+package Cx
+
+	#search for harcoded secrets by looking for their values with a special chars and length
+CxPolicy [ result ] {
```

Comment:
Do you think we can use strategy number 1. and 2. described here https://www.fugue.co/blog/5-tips-for-using-the-rego-language-for-open-policy-agent-opa to reduce the amount of **OR** statements inside this policy file.